.github/workflows/build.yaml

```
@@ -0,0 +1,23 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+name: Java CI with Maven
+on: push
```

It might be worth trying the following:
```suggestion
on: [push, pull_request]
```

Not 100% sure that it will do what we want, but its a possibility?

---

That worked, thanks.